android/KMEA/app/src/main/java/com/tavultesoft/kmea/data/CloudApiTypes.java

```diff
@@ -33,25 +35,44 @@ public CloudApiReturns(ApiTarget target, JSONObject jsonObject) {
 
   }
 
-  protected enum ApiTarget {
+  public enum ApiTarget {
     Keyboards,
-    LexicalModels
+    LexicalModels,
+    Keyboard,
+    KeyBoardLexicalModels,
+    KeyboardData,
+    LexicalModelPackage,
```

Comment:
1. Can you lower-case the 'b' in `KeyBoardLexicalModels`?

2. Please add some notes how these ApiTargets get used: 
(for instance)
* Keyboards vs Keyboard
* LexicalModels vs KeyBoardLexicalModels vs LexicalModelPackage